### PR TITLE
[#138322855] - add 'start supplier declaration' page

### PR DESCRIPTION
**Pull request by samuelhwilliams**
Story: https://www.pivotaltracker.com/story/show/138322855

This PR inserts a new page into the supplier declaration journey, signposting what they are about to do and why.

screen shot 2017-01-26 at 12 41 15

### CODE REVIEW

File: app/main/views/frameworks.py [hunk 1]
```diff
@@ -259,6 +259,17 @@ def framework_submission_services(framework_slug, lot_slug):
     ), 200
 
 
+@main.route('/frameworks/<framework_slug>/start-declaration', methods=['GET'])
```
Comment (TheDoubleK) [inline app/main/views/frameworks.py]: Of course! Nice URL.

File: app/templates/frameworks/_framework_actions.html [hunk 2]
```diff
@@ -25,7 +25,9 @@
 
 {% if framework.status == 'open' %}
 <li class="browse-list-item">
-  <a class="browse-list-item-link" href="{{ url_for('.framework_supplier_declaration', framework_slug=framework.slug, section_id=first_page_of_declaration) }}">
+  <a class="browse-list-item-link" href="{{ url_for('.framework_start_supplier_declaration', framework_slug=framework.slug)
```
Comment (TheDoubleK) [inline app/templates/frameworks/_framework_actions.html]: It's might be clearer overall to roll the conditional href and anchor text into one `declaration_status` check here? Something like 
```
{% if declaration_status == 'unstarted' %}
  <a href=your new page>Make supplier declaration</a>
(% else %}
  <a href=old link>Edit supplier declaration</a>
{% endif %}
```

File: app/templates/frameworks/start_declaration.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+         smaller = True
+      %}
+        {% include "toolkit/page-heading.html" %}
+      {% endwith %}
+
+      <p>You must answer all questions in the declaration to confirm you meet the minimum standards for providing digital services to government.</p>
+      <br>
+      <p>The Crown Commercial Service will review your answers. If you don’t meet the minimum standards, your application won’t be successful.</p>
+      <br>
+      <p>You can come back and change your answers at any time before the framework application deadline, 5pm BST, 23 June 2016.</p>
```
Comment (TheDoubleK) [inline app/templates/frameworks/start_declaration.html]: The date here should come from the `framework_close_date` in the framework content, which you should be able to get at in the view the same time you get the "next section" name (have a look at existing code, you might need to `load_messages('dates')` or something like that).

**Comment by samuelhwilliams** [comment]
Added tests and made changes per feedback

**Comment by TheDoubleK** [comment]
Looking good, although it needs: 
 - Appease PEP8
 - Rebase
 - Wait for Travis tests to pass green
 - :+1: merge

**Comment by samuelhwilliams** [comment]
Changes made and tests pass.

The content on this new page has not been finalised yet, so assume can't merge/release until that's signed off?

**Comment by TheDoubleK** [comment]
I think we should merge it now - content changes will come in as "tweak" stories later.  More important to get the full flow functioning so people can see it - makes it much easier for people to suggest content changes when they can see it.
:+1:

**Pull request merged**